db/sample_data.rb

```diff
@@ -67,8 +68,7 @@ def add_sample_data
   [events, users, profiles, application_letters, requests, agreement_letters].each do |models|
     save_models(models)
   end
-
-  # set deadline to past to work around validation of application letters
```

Comment:
Why did you remove this comment? I think it's important to understand why we already save things here.